core/modules/admin_bar/tests/admin_bar.test

```diff
@@ -490,4 +490,60 @@ class AdminBarCustomizedTestCase extends AdminBarWebTestCase {
       ':path' => $path,
     ))->fetchField();
   }
+  /**
+   * Test the "Back To Site" link functionality.
+   *
+   * @broken
+   * @todo Figure out how to test for HTML in the admin bar. The debug output for
+   *   the failed tests doesn't seem to give me that information.
+   */
+  /*
+  function testEscapeAdmin() {
+
+    // Create a sample page.
+    $settings = array(
+      'title'     => 'Foo',
+    );
+    $this->backdropCreateNode($settings);
+
+    // Set theme to something other than Stark.
+    config_set('system.core', 'theme_default', 'basis');
+    config_set('system.core', 'admin_theme', 'seven');
+
+    // Login user and go to created page.
+    $permissions = $this->basePermissions + array(
+        'access content overview',
+        'view the administration theme'
+      );
+    $this->web_user = $this->backdropCreateUser($permissions);
+    $this->backdropLogin($this->web_user);
+    $this->backdropGet('foo');
+
+    // Go to an admin page and see that the link has not changed.
+    // Using the raw text method since the link is in the HTML and not visible.
+    $this->backdropGet('admin/content');
+    $this->assertRaw('<li class="admin-bar-icon expandable"><a href="/" class="">');
+
+    // Set the escape admin variable.
+    config_set('admin_bar.settings', 'back_to_site', 1);
+
+    // Clear the cache to make sure variable change takes effect.
+    //cache_clear_all();
+
+    // Go back to non-admin page and then back to admin page.
+    $this->backdropGet('foo');
+    $this->backdropGet('admin/content');
+
+    // See that the "Back To Site" link has been added.
+    $this->assertRaw('<li class="admin-bar-icon expandable"><a href="/node/2"');
+
+    // Go to the front page and then back to the admin.
+    $front = config_get('system.core', 'site_frontpage');
+    $this->backdropGet($front);
+    $this->backdropGet('admin/content');
+
+    // See that the Home link is still there.
+    $this->assertRaw('<li class="admin-bar-icon expandable"><a href="/" class="">');
+  }
+  */
```

Comment:
Remove this too.

Reply:
All the comments and suggestions for the test are moot, as the whole test needs to be removed.